### ISSUE TITLE
Extension broken since Firefox 72

### ISSUE THREAD

**Issue by iamphpgo** [pedrocr/proxyswitcheroo#5]
Hello 

ProxySwitcheroo is not working on the new Firefox 72.0.1 version on ubuntu 18.04.

**Comment by pedrocr** [comment]
I can replicate that on 19.10 as well. The console shows the following error message:

```
Error: respectBeConservative can be set by privileged extensions only. ExtensionUtils.jsm
```

I'm trying to figure out what that is.

**Comment by pedrocr** [comment]
I may have done something wrong but I suspect a Firefox bug so I've filed it:

https://bugzilla.mozilla.org/show_bug.cgi?id=1608373

**Comment by pedrocr** [comment]
I've released 1.0.1 with a workaround. It's now available in the extension archive, updating Firefox extensions should be all that's needed.
